app/app.py

```diff
@@ -2,28 +2,44 @@
 import logging
 import os
 from pathlib import Path
-from subprocess import check_call
+from http import HTTPStatus
 
 from flask import Flask, request, abort
 
+from .commands import ConnectTo, NotConfigured
+
+# Add/change the deployments here!
+DEPLOYMENTS = {
+    "gunaha": NotConfigured(),
+    "hello": ConnectTo("itw.altlab.dev").command("/opt/hello.altlab.dev/deploy"),
```

Comment:
We might want do doucment and standardized where our applications will live. For now, somewhere in `/opt/` is good.